view/topbar.twig

```diff
@@ -1,4 +1,20 @@
-<a {% if request.vocabid == '' and request.page != 'feedback' and request.page != 'about' %}id="service-logo"{% endif %} {% if request.vocabid != '' or request.page == 'feedback' or request.page == 'about' %}class="service-{{ request.lang }}" {% endif %}href="{{ request.lang }}/{% if request.contentLang != request.lang %}?clang={{ request.contentLang }}{% endif %}{% if request.queryParam('anylang') == 'on' %}{% if request.contentLang == request.lang %}?{% else %}&{% endif %}anylang=on{% endif %}"><h1 id="service-name">{% if ServiceName is defined  %}{{ ServiceName }}{% else %}Skosmos{% endif %}</h1></a>
+<a
```

Comment:
I wonder what has been changed at this point that causes such conflict. Anyways, we should have a proper conversation about Twig filters/functions (and yes, even if they are in defined in PHP code, they are still part of Twig which is - of course - coded in PHP). But I agree to some point. However, I am not sure if this is the PR in which to fix 'em.

Reply:
Yes, I think it will be much easier to write this as a proper Twig filter/function (in PHP) now that it is refactored. :) That would simplify the template quite a bit, not to forget the availability of PHP functions there..

About the (merge) conflict: I was just wondering who had changed this part of the code because I could not remember seeing any changes here lately - it was later revealed that it was me (https://github.com/NatLibFi/Skosmos/blob/master/view/topbar.twig) :D